packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -115,6 +119,21 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
           : 20
         : this.props.margins?.left ?? 40,
     };
+    if (this.props.xAxisTitle !== undefined && this.props.xAxisTitle !== '') {
+      this.margins.bottom! = this.props.margins?.bottom ?? 70;
```

Comment:
shouldn't this be margin.bottom + some X value

Reply:
same for y and secondary y as well.

Reply:
kept it fixed values to ensure we do not add up more values on top of the existing ones leading to erroneous values.